PWG/EMCAL/config/AliEmcalCorrectionConfiguration.yaml

```diff
@@ -149,6 +149,7 @@ ClusterNonLinearity:                                # Cluster non-linearity corr
     enabled: false                                  # Whether to enable the task
     createHistos: false                             # Whether the task should create output histograms
     nonLinFunct: kBeamTestCorrectedv3               # Sets the non-linearity function. Possible options are defined in the header of the cluster non-linearity correction component
+    setClusterE: false                              # Modify the cluster energy data member
```

Comment:
We suggest naming this `setForceClusterE`, and I would suggest making this clear that it is for backwards compatibility and not generally recommended. Perhaps the comment could be:

```
# Only for backwards compatibility, force cluster->E() to be set to the cluster non-linearity corrected energy. Off by default. For the standard methods, see: http://alidoc.cern.ch/AliPhysics/master/READMEcontainers.html#emcalContainerClusterEnergyCorrections
```